knockout-tinymce.coffee

```diff
@@ -1,4 +1,5 @@
 (($, ko) ->
+  cache = ""
```

Comment:
Since the scope of the `cache` variable is outside the scope of the `init` function, only one `cache` variable will be used for all bound tinymce editors. Perhaps we should use a [`WeakMap`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WeakMap) with `element` as the keys?